decko/decko.gemspec

```diff
@@ -34,7 +34,6 @@ Gem::Specification.new do |s|
   s.required_ruby_version = ">= 2.3"
 
   [
-    ["rails", "~> 5.1"],
```

Comment:
because it's in card?

Reply:
y, I moved it to card and changed it to < 5.2